models/fetchAllArticles.model.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../db/connection.js')
+
+function fetchAllArticles(query) {
+    let sql = "SELECT article_id, title, topic, author, created_at, votes, article_img_url FROM articles"
+
+    if(query) {
+        sql += " WHERE topic = $1 ORDER BY created_at DESC"
+
```

Comment:
You can tidy this up by conditionally pushing the query to the array. This will make the code DRY-er as we'll only need one db.query